src/agent/atexit/Cargo.toml

```diff
@@ -6,6 +6,6 @@ authors = ["fuzzing@microsoft.com"]
 edition = "2018"
 
 [dependencies]
-ctrlc = "3.2"
+ctrlc = "3.2.3"
```

Comment:
What there a reason you needed updated the lower bound here, and not just in the lockfile? If so, it may be good to document.

Reply:
Not for any particular reason. It feels a little more repeatable to me to do it this way, but perhaps that’s superstitious 🙂